surefire-provider/src/main/java/org/arquillian/smart/testing/surefire/provider/TestStrategyApplier.java

```diff
@@ -85,4 +92,25 @@ private boolean presentOnClasspath(String testClass) {
             throw new IllegalStateException(e);
         }
     }
+
+    private Collection<TestSelection> filterMergeAndOrderTestSelection(Collection<TestSelection> selectedTests,
+        List<String> strategies) {
+
+        final Collection<TestSelection> testSelections = selectedTests
+            .stream()
+            .filter(testSelection -> presentOnClasspath(testSelection.getClassName()))
```

Comment:
this is also checked here https://github.com/arquillian/smart-testing/pull/119/files#diff-356cfd962197d56c7e69c381ba349281R65